core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.properties

```diff
@@ -0,0 +1,2 @@
+Back\ to\ Dashboard=Back to Dashboard
```

Comment:
Why wut? You mean this file isn't even necessary cause it's defined by default in the corresponding files? (Cannot check now on the phone).